test/e2e/dynamic_provisioning_test.go

```diff
@@ -1286,4 +1286,60 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool, schedulerNa
 		}
 		test.Run(cs, ns, schedulerName)
 	})
+	ginkgo.It("Laila Should test an increase in replicas when scaling up", func() {
```

Comment:
Planning to commit yourself to the repository to run the tests? 😆

Reply:
Ope guess I'll have to leave my legacy elsewhere